app/assets/stylesheets/targets.scss

```diff
@@ -15,7 +15,7 @@
   display: inline-block;
 }
 
-table , td{
+.table , .table td{
```

Comment:
Be careful with the names of the classes here. Remember that `table` is the name of an HTML tag. Names of the classes should have a semantic meaning, that way the code will be self explanatory and better to understand. For example:

```
.c-header-logo {
  /* Just by reading the name, I guess this selector targets the logo of the header. */
}
```